mds/apis/prv_api/vehicles.py

```diff
@@ -89,10 +91,14 @@ class Meta:
         )
 
 
-class RetrieveDeviceSerializer(BaseDeviceSerializer):
+class RetrieveDeviceSerializer(DeviceSerializer):
     areas = serializers.SerializerMethodField()
+    provider_logo = serializers.CharField(
+        source="provider.logo_b64",
+        help_text="logo of the service provider of the device",
+    )
 
-    def get_areas(self, obj):
+    def get_areas(self, obj) -> List[Dict[str, str]]:
```

Comment:
Does this help with schema generation ? If yes, that's good to know

Reply:
yes, it does.

Reply:
It seems to be an alternative solution to swagger decoration